src/content/en/updates/2018/04/loading-wasm.md

```diff
@@ -31,7 +31,7 @@ This code snippet does the complete download-compile-instantiate dance, albeit i
   const buffer = await response.arrayBuffer();
   const module = new WebAssembly.Module(buffer);
   const instance = new WebAssembly.Instance(module);
-  const result = instance.exports.fibonacci(42);
+  console.log(instance.exports.fibonacci(42));
```

Comment:
Too much is happening on this one line, IMHO. I actually prefer this the way it was before. If you want to add a `console.log`, can we just do `console.log(result)` please?

Reply:
Same for the other instances below

Reply:
These changes has been pushed live already. I'll address your feedback in a separate CL.

Reply:
Please do! Thanks.

Reply:
See https://github.com/google/WebFundamentals/pull/6043